l10n_it_location_nuts/tests/test_nuts.py

```diff
@@ -2,10 +2,10 @@
 # Copyright 2022 Simone Rubino - TAKOBI
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from odoo.tests.common import SavepointCase
```

Comment:
questa parte dovrebbe essere nel commit di migrazione [[MIG] l10n_it_location_nuts: Migration to 16.0](https://github.com/OCA/l10n-italy/pull/3029/commits/f7243bfb83ae0b648041c65a60e25a88e3abafef)